Ruddarr/Views/Series/SeriesContextMenu.swift

```diff
@@ -1,19 +1,42 @@
 import SwiftUI
+import TelemetryDeck
 
 struct SeriesContextMenu: View {
     var series: Series
+    @Environment(SonarrInstance.self) private var instance
 
     var body: some View {
         link(name: "Trakt", url: traktUrl)
         link(name: "IMDb", url: imdbUrl)
         link(name: "TVDB", url: tvdbUrl)
+        
+        Divider()
+        
+        Button("Search Monitored", systemImage: "magnifyingglass") {
+            Task { await dispatchSearch() }
+        }
+        .disabled(!series.monitored)
```

Comment:
Maybe we should hide the button if it's not monitored?

Reply:
Solid point. Happy to make that change this evening